psh/ls/ls.c

```diff
@@ -190,6 +196,27 @@ static int psh_ls_copyname(fileinfo_t *file, const char *name)
 }
 
 
+static int psh_ls_statentry(fileinfo_t *file, const char *path, bool followSymbolic)
+{
+	int ret;
+	struct stat st;
+
+	ret = followSymbolic ? lstat(path, &st) : stat(path, &st);
```

Comment:
The name `followSymbolic` is confusing. Iit feels like the logic should be inverted since `lstat` doesn't follow symlinks. Maybe rename it to `resolveSymlink` and invert the logic?